content/v2.0/influxdb-templates/create.md

```diff
@@ -111,6 +111,70 @@ influx pkg export all \
   --telegraf-configs=00000x0x000X0x0X0
 ```
 
+## Include user-definable resource names
```

Comment:
when can this task happen? Perhaps link(s) in overview section where it makes sense...